Package.swift

```diff
@@ -15,12 +15,12 @@ let package = Package(
             targets: ["WhisperKit"]
         ),
         .executable(
-            name: "transcribe",
+            name: "whisper-kit",
```

Comment:
this is a bit unfortunate, looks like we cannot have `whisperkit` executable and `WhisperKit` module because the whole thing is not compiling, @ZachNagengast please advise